classes/class-dintero-checkout-widget.php

```diff
@@ -67,7 +67,7 @@ public function form( $instance ) {
 		</p>
 		<p>
 			<label for="<?php echo esc_attr( $icon_color ); ?>"><?php esc_html_e( 'Icon color:', 'dintero-checkout-for-woocommerce' ); ?>
-			<input type="color" class="widefat colorpick" value="#cecece" id="<?php echo esc_attr( $icon_color ); ?>" name="<?php echo esc_attr( $icon_color ); ?>"  /></label>	
+			<input type="color" class="widefat colorpick" value="#cecece" id="<?php echo esc_attr( $icon_color ); ?>" name="<?php echo esc_attr( $icon_color ); ?>"  /></label>
```

Comment:
I'm not sure what these changes are. Maybe IntelliJ auto-fixing some syntax.

Reply:
I've made some adjustments to the formatting.